test/filters/filter.spec.js

```diff
@@ -19,8 +18,6 @@ describe('Directive:  pfFilter', function () {
     var el = $compile(element)(scope);
 
     scope.$digest();
-
-    isoScope = el.isolateScope();
```

Comment:
Note: I've had success replace isoScope with the following:

```diff
-    isoScope = el.isolateScope();
+    isoScope = el.controller('pfDonutPctChart');
```
Where 'pfDonutPctChart' is the name of the component controller your after.